carthage-software/mago/0.24/mago.toml

```diff
@@ -0,0 +1,22 @@
+php_version = "8.2"
+
+[source]
+paths = [
+  "src/",
+  "config/",
+  "bin/",
+  # "tests/",
+  # "importmap.php",
+  # "migrations/",
+]
+includes = ["vendor"]
+excludes = []
+
+[format]
+print_width = 120
+tab_width = 4
+use_tabs = false
```

Comment:
Those are indeed the defaults, `mago init` generates this just to make it obvious where formatting settings go, it could be removed, or commented out with a link to the formatter docs? ref: https://docs.rs/mago-formatter/latest/mago_formatter/settings/struct.FormatSettings.html